### DISCUSSION TITLE
about "Pin scripts tab #17160"

### DISCUSSION THREAD

**Discussion by podoni** [dbeaver/dbeaver#20473] [Q&A] [answered]
Pin scripts tab  #17160

Has the development schedule been canceled due to the "Pin scripts tab #17160" issue?
It was confirmed that it was deleted at 23.1.1 milestone.

If it is possible to implement a function, it would be very useful.

Thank for Reading. :)

**Comment by E1izabeth** [comment, marked as answer]
We made an investigation and it turns out that implementation is quite time-expensive, so it's postponed. We want to know how many people want this feature to understand does it worth. You can vote for the issue with 👍🏻 under the issue description.
